libyui-qt/src/YQMainWinDock.cc

```diff
@@ -157,6 +150,36 @@ YQMainWinDock::add( YQDialog * dialog )
 }
 
 
+void
+YQMainWinDock::raiseDialog( YQDialog * newCurrentDialog )
+{
+    if ( ! newCurrentDialog )
+        return;
+
+    newCurrentDialog->raise();
+    newCurrentDialog->show();
+    newCurrentDialog->update();
```

Comment:
Shouldn't be necessary, but also shouldn't hurt either

Reply:
IIRC multiple `update()` calls only set a flag for a later update.

Reply:
Yep! And I'd hope that either `raise()` or `show()` already trigger an update event...